doc/unity.rst

```diff
@@ -10,6 +10,9 @@ runtime to use the MuJoCo physics engine.  Users can import MJCF files and edit
 relies on Unity for most aspects -- assets, game logic, simulation time -- but uses MuJoCo to determine how objects
 move, giving the designer access to MuJoCo's full API.
 
+An example project using MuJoCo's Unity plugin in a set of introductory tutorials is available at
+https://github.com/Balint-H/mj-unity-tutorial.
```

Comment:
please make this a proper RST link

Reply:
There are a couple other points in the original file that uses direct urls, should I swap those as well?

Reply:
Yes please!

Reply:
Better late than never, finally added this switch.

Reply:
Can you remove the trailing whitespaces in unity.rst? There are 3, I believe

Reply:
My bad, removed.